### PR TITLE
Update to fricas 1.3.4

### PR DESCRIPTION
**Pull request by pietvo**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.x
Xcode 8.x

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: math/fricas/Portfile [hunk 1]
```diff
@@ -28,10 +28,10 @@ distname            ${name}-${version}-full
...
 worksrcdir          ${name}-${version}
                 
-checksums           rmd160  1ab5c148f0684c946c3b0831a5fe01fc21bbe83a \
-                    sha256  798b123e61f3566e6b1b367c235f5b4599a9c737b938aa71f32d0aabb7db439d \
-                    md5     0d3af65758ce93b1cc52c2511e73e674 \
-                    sha1    8fc3e850a9890eac21cd9f391dd58000b3537067
+checksums           rmd160  10aac421e8100668b2e6a9a598da353290ca354e \
+                    sha256  fd94bb0ff5f4de1dc10eef37446a6e7b5bf14d61852089002a7e0813be4ad394 \
+                    md5     cef44e511d124b1f6e1f6b5ac53fd496 \
+                    sha1    3e84e978fd5ba0b269890b7acfdad9ba3d380c31
```
Comment (pmetzger) [inline math/fricas/Portfile]: We no longer include md5 or sha1 hashes in our Portfiles. However, we now _do_ include a `size` field. If you can add that, and remove the obsolete hashes, I'll commit this very quickly.

**Comment by pietvo** [comment]
Requested changes applied.

**Comment by pietvo** [comment]
Perry E. Metzger wrote:

 > @pmetzger requested changes on this pull request.
 > 
 > --------------------------------------------------------------------------------------------------
 > 
 > In math/fricas/Portfile:
 > 
 > > @@ -28,10 +28,10 @@ distname            ${name}-${version}-full
 >  use_bzip2           yes
 >  worksrcdir          ${name}-${version}
 >                  
 > -checksums           rmd160  1ab5c148f0684c946c3b0831a5fe01fc21bbe83a \
 > -                    sha256  798b123e61f3566e6b1b367c235f5b4599a9c737b938aa71f32d0aabb7db439d \
 > -                    md5     0d3af65758ce93b1cc52c2511e73e674 \
 > -                    sha1    8fc3e850a9890eac21cd9f391dd58000b3537067
 > +checksums           rmd160  10aac421e8100668b2e6a9a598da353290ca354e \
 > +                    sha256  fd94bb0ff5f4de1dc10eef37446a6e7b5bf14d61852089002a7e0813be4ad394 \
 > +                    md5     cef44e511d124b1f6e1f6b5ac53fd496 \
 > +                    sha1    3e84e978fd5ba0b269890b7acfdad9ba3d380c31
 > 
 > We no longer include md5 or sha1 hashes in our Portfiles. However, we now do include a size field.
 > If you can add that, and remove the obsolete hashes, I'll commit this very quickly.
 > 
 > —
 > You are receiving this because you authored the thread.
 > Reply to this email directly, view it on GitHub, or mute the thread.*
 > 

I have applied the requested changes. Will this suffice, or do I have to create a new pull request?
I am not very familiar with the pull request business.

With kind regards,
-- 
Piet van Oostrum <piet@vanoostrum.org>
WWW: http://piet.vanoostrum.org/
PGP key: [8DAE142BE17999C4]

**Comment by pmetzger** [comment]
Hi @pietvo! There are now merge conflicts. I would suggest that you rebase. That will also have the effect of squishing your changes, which we require (we usually only want a single commit for something like this.) After you do that, we can merge your update.

**Comment by pietvo** [comment]
Perry E. Metzger wrote:

 > Hi @pietvo! There are now merge conflicts. I would suggest that you rebase. That will also have
 > the effect of squishing your changes, which we require (we usually only want a single commit for
 > something like this.) After you do that, we can merge your update.
 > 
 > —
 > You are receiving this because you were mentioned.
 > Reply to this email directly, view it on GitHub, or mute the thread.*
 > 

Hi Perry,

Unfortunately I did not make my changes on a separate branch, but on the master branch. Which in hindsight was a bit silly.
Now I am at a loss how I should do the rebase solely on the master branch, in such a way that my fork ends up with only a single commit. Can you help me out?

With kind regards,
-- 
Piet van Oostrum <piet@vanoostrum.org>
WWW: http://piet.vanoostrum.org/
PGP key: [8DAE142BE17999C4]

**Comment by mf2k** [comment]
You can always just make a copy of your 2 modified files, revert your repository to the latest version and start over. Then replace the 2 files.

**Comment by pmetzger** [comment]
If you do switch to a new branch, you'll need to open a new PR and close this one, but that's no problem.

Another alternative (copy those files to the side first!) is to do a `git reset --soft HEAD^^^` or some such, unstage the commits, stash them, branch, unstash, commit again.

**Comment by pietvo** [comment]
I Will make a new pull request.

**Pull request closed without merging**